spek-runner/junit5/src/main/kotlin/org/spekframework/spek2/junit/SpekTestDescriptor.kt

```diff
@@ -70,7 +69,7 @@ class SpekTestDescriptor internal constructor(
 
     override fun getChildren() = childDescriptors
 
-    override fun mayRegisterTests(): Boolean = scope is ActionScopeImpl
+    override fun mayRegisterTests(): Boolean = false
```

Comment:
We can omit this since the default implementation returns `false`.

Reply:
Nice! Done.